plugins/auth-node/src/DefaultIdentityClient.ts

```diff
@@ -78,17 +78,32 @@ export class DefaultIdentityClient implements IdentityApi {
       : ['ES256'];
   }
 
-  async getIdentity({ request }: IdentityApiGetIdentityRequest) {
+  async getIdentity({
+    request,
+    optional = false,
+  }: IdentityApiGetIdentityRequest): Promise<
+    BackstageIdentityResponse | undefined
```

Comment:
We wouldn't return `undefined` if optional is false. could we perhaps overload this method to simplify the return signature slightly.

Reply:
I'm sorry, what do you mean by overloading this method ?